### PR TITLE
Go dependency updates

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: pkg/preflight/builtin_test.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/replicatedhq/kurl/pkg/installer"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	yaml "gopkg.in/yaml.v3"
+	yaml "gopkg.in/yaml.v2"
```
Comment (laverya) [inline pkg/preflight/builtin_test.go:13]: why the downgrade?

Comment (laverya) [reply pkg/preflight/builtin_test.go:13]: ooh because it was the only v3 usage, ok

**Review by laverya — APPROVED** [review]
Looks good, but I want to see this in testgrid before we release just in case the yaml library version change breaks things

**Comment by emosbaugh** [comment]
Damn, builds are failing. I ran it locally and it worked. Ill have to see what is the difference.

File: pkg/preflight/builtin_test.go (hunk 1 above)
Comment (emosbaugh) [reply pkg/preflight/builtin_test.go:13]: had to add it back anyway. unmarshal failed

File: kurl_util/cmd/yamltobash/main.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ import (
 	kurlscheme "github.com/replicatedhq/kurl/kurlkinds/client/kurlclientset/scheme"
 	kurlv1beta1 "github.com/replicatedhq/kurl/kurlkinds/pkg/apis/cluster/v1beta1"
 	kurlversion "github.com/replicatedhq/kurl/pkg/version"
-	"gopkg.in/yaml.v1"
```
Comment (emosbaugh) [inline kurl_util/cmd/yamltobash/main.go:19]: @areed is there any reason this is using v1?

Comment (areed) [reply kurl_util/cmd/yamltobash/main.go:19]: Not that I'm aware of

**Review by laverya — APPROVED** [review]

**Pull request merged**